package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@uma/react-library",
+  "name": "@uma/react",
```

Comment:
can we name this to just `@uma/react`? having `library` feels redundant since it will be an npm module

Reply:
+1

Reply:
Absolutely fine by me.